app/helpers/comparison_helper.rb

```diff
@@ -0,0 +1,12 @@
+module ComparisonHelper
```

Comment:
The `ComparisonHelper` module has been added to prepare slugs. Ensure that the `prepare_slugs` method properly sanitizes and validates the input to prevent potential security issues.